jwst/tweakreg/astrometric_utils.py

```diff
@@ -81,8 +87,15 @@ def create_astrometric_catalog(input_models, catalog="GAIADR3", output="ref_cat.
     radius, fiducial = compute_radius(outwcs)
 
     # perform query for this field-of-view
-    ref_dict = get_catalog(fiducial[0], fiducial[1], sr=radius, catalog=catalog)
-    colnames = ('ra', 'dec', 'mag', 'objID')
+    epoch = (
+        epoch
+        if epoch is not None
+        else Time(input_models[0].meta.observation.date).decimalyear
+    )
+    # keep only decimal point and digit characters
+    epoch = float("".join(c for c in str(epoch) if c == "." or c.isdigit()))
```

Comment:
Not sure this line is needed anymore, since the result of line 93 is already a simple decimal number (e.g. 2023.476), but it certainly can't hurt to keep it there even if it does nothing.

Reply:
Good call. I got rid of the line and forced epoch to be an optional float, as opposed to an optional float or string.